enterprise/models.py

```diff
@@ -4338,9 +4338,8 @@ def _get_explicit_group_members(self, user_query=None, fetch_removed=False, pend
         """
         Fetch explicitly defined members of a group, indicated by an existing membership record
         """
-        members = self.members.select_related(
-            'enterprise_customer_user', 'pending_enterprise_customer_user'
-        )
+        members = EnterpriseGroupMembership.all_objects.filter(
```

Comment:
might be worth a comment about how reverse lookups wouldn't surface soft deleted items?